tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -68,6 +68,20 @@ module.exports = {
         .waitForElementVisible(actionLocatorButton)
         .click(actionLocatorButton)
         .waitForOutstandingAjaxCalls()
+    },
+    /**
+     * @param {string} element
+     * @param {string} sharer
+     *
+     *Asserts that the element(file/folder/resource) on the shared-with-me page is shared by the desired user
+     */
+    assertSharedByDesiredUser: function (element, sharer) {
```

Comment:
`Desired` is not necessary here.
```suggestion
    assertSharedByUser: function (element, sharer) {
```